libtesting-ui/src/main/java/com/mapbox/navigation/testing/ui/http/MockWebServerRule.kt

```diff
@@ -64,4 +73,18 @@ class MockWebServerRule : TestWatcher() {
         requestHandlers.clear()
         webServer.shutdown()
     }
+
+    suspend fun withoutWebServer(block: suspend () -> Unit) {
+        val previousPort = webServer.port
+        webServer.shutdown()
+        try {
+            block()
+        } finally {
+            withContext(Dispatchers.IO) {
+                webServer = MockWebServer()
+                initDispatcher()
+                webServer.start(previousPort)
```

Comment:
There is a chance that the port won't be available. We will need to find a different strategy if tests start fail because of this